Makefile

```diff
@@ -258,28 +266,27 @@ GLOBAL_ASM_RACING_O_FILES = $(foreach file,$(GLOBAL_ASM_RACING_C_FILES),$(BUILD_
 # Compiler Options                                                             #
 #==============================================================================#
 
+# detect iconv
+ICONV ?= iconv
+
 # detect prefix for MIPS toolchain
-ifneq      ($(call find-command,mips-linux-gnu-ld),)
-  CROSS := mips-linux-gnu-
+ifneq ($(CROSS),)
```

Comment:
Is this necessary versus the old way?

ifneq will always evaluate to true because cross hasn't been set to anything yet?

Reply:
ifneq ($(CROSS),) are always false because Cross are equal to nothing and like this we allow make CROSS=my-cross for example because it's a thing who change for each distro